setup.cfg

```
@@ -39,7 +39,7 @@ tag_prefix =
 parentdir_prefix = distributed-
 
 [tool:pytest]
-addopts = -v -rsxfE --durations=20 --color=yes --ignore=continuous_integration --ignore=docs --ignore=.github --strict-markers --strict-config
+addopts = -v -rsxfE --durations=20 --color=yes --ignore=continuous_integration --ignore=docs --ignore=.github --strict-markers --strict-config -p no:asyncio
```

nit: I would suggest to use a line per option. This makes readability better and provides better diffs

We can of course do this in a follow up PR
```suggestion
addopts =
    -v
    -rsxfE
    --durations=20
    --color=yes
    --ignore=continuous_integration
    --ignore=docs
    --ignore=.github
    --strict-markers
    --strict-config
    -p no:asyncio
```

---

Good idea!